PP_APP/server/frontend/templates/post_view.html

```diff
@@ -0,0 +1,59 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+
+<section id="about" class="about-section bg-light">
+    <div class="container">
+        {% for key in pp_post %}
+            <h1>POST #{{key.img_post.id}}</h1>
+            <table class="table">
+                <tr>
+                    <th>User</th>
+                    <td>{{key.img_post.user_id.email}}</td>
+                </tr>
+                <tr>
+                    <th>Top</th>
```

Comment:
Great page. After the beta we can actually display these images instead of providing a link.